server/scripts/import-tests/populate-test-data.js

```diff
@@ -0,0 +1,19 @@
+/* eslint-disable no-console */
+const fs = require('fs');
+const db = require('../../models');
+
+const populateTestDatabase = async () => {
+    const testDataScript = fs.readFileSync(
+        __dirname + '/pg_dump_2021_05_test_data.sql',
+        'utf-8'
+    );
+    console.info('test.data.dump', testDataScript);
+    return await db.sequelize.query(testDataScript);
+};
+
+if (require.main === module)
+    populateTestDatabase()
+        .then(() => console.info('Successfully Populated'))
+        .finally(() => console.info('Test Data Populated'));
```

Comment:
I don't think we need this `.finally` (?) And the fact it will say "test data populated" when errors occurred seems like it might be confusing!

Reply:
Good point!